### ISSUE TITLE
Errors with included executable file

### ISSUE THREAD

**Issue by tomplex** [joelibaceta/video-to-ascii#8]
Installing on macos Mojave using the system pip and python yields the following error on execution:

```bash
❯ video-to-ascii
/usr/local/bin/video-to-ascii: line 3: import: command not found
from: can't read /var/mail/video_to_ascii
/usr/local/bin/video-to-ascii: line 6: syntax error near unexpected token `('
/usr/local/bin/video-to-ascii: line 6: `CLI_DESC = ("It is a simple python package to play videos in the terminal"'
```

On a closer look, it's because the file [bin/video-to-ascii](bin/video-to-ascii) is missing the `!` from the shebang line, which should read like so:

```bash
#!/usr/bin/env python
```

Furthermore, you'll also want to include a line like this:

```bash
# coding=utf-8
```

Otherwise, I get this error:

```bash
258 ❯ video-to-ascii
  File "/usr/local/lib/python2.7/bin/video-to-ascii", line 9
SyntaxError: Non-ASCII character '\xf0' in file /usr/local/lib/python2.7/bin/video-to-ascii on line 9, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

**Comment by joelibaceta** [comment]
Thanks @tomplex I'm fixing this.

PS: Maybe you might need python 3

**Comment by joelibaceta** [comment]
Solved in https://github.com/joelibaceta/video-to-ascii/commit/077161b152d10d58a6da33ee87f880b5e1f49537

**Comment by joelibaceta** [comment]
Available in 1.1.8 version https://pypi.org/project/video-to-ascii/1.1.8/

**Comment by tomplex** [comment]
Thanks for the quick fix!
